### ISSUE TITLE
Add-PartnerCustomerUserRoleMember doesn't work

### ISSUE THREAD

**Issue by sirovermark** [Microsoft/Partner-Center-PowerShell#90]
# Steps to reproduce

```
Add-PartnerCustomerUserRoleMember -CustomerId $CustomerId -UserId '$UserId -RoleId $RoleId
Add-PartnerCustomerUserRoleMember : 2000 Customer id is required.
At line:1 char:9
+ Add-PartnerCustomerUserRoleMember -CustomerId 'xxxxxxx-xxxxxxxx ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-PartnerCustomerUserRoleMember], PartnerPSException
    + FullyQualifiedErrorId : Microsoft.Store.PartnerCenter.PowerShell.Exceptions.PartnerPSException,Microsoft.Store.PartnerCenter.PowerShell.Commands.AddPartnerCustomerUserRoleMember
```

## Actual behavior
It throws an error because CustomerId even it's ok.

## Diagnostic logs

> Please share test platform diagnostics logs.  
> The logs may contain test assembly paths, kindly review and mask those before sharing.

## Environment

```
Get-Module PartnerCenter | select Name,Version

Name          Version   
----          -------   
PartnerCenter 1.5.1902.5
```

![image](https://user-images.githubusercontent.com/30412009/53565588-27108400-3b5a-11e9-8e33-0ec68f417ba0.png)

**Comment by isaiahwilliams** [comment]
Hi @sirovermark, 

I have identified what is causing this issue and it will be resolved in the next release. We are targeting the end of next week for the next release. Please let us know if you find any other issues.

**Comment by sirovermark** [comment]
Hi @isaiahwilliams,

Great to know. Thank you!

**Comment by isaiahwilliams** [comment]
Hi @sirovermark, 

The required work to address this issue has been completed. Right now the plan is to release a new version of the module on March 15th. Once that is done, you will be able to update the module and everything will function as expected. Please let us know if you discover any other issues. We greatly appreciate you reporting this!

**Comment by isaiahwilliams** [comment]
Hi @sirovermar,

We just released version 1.5.1903.5 which includes a fix for this issue. Please upgrade and let us know if you have any issues.

**Comment by sirovermark** [comment]
Thanks @isaiahwilliams for the effort!
